plugins/pulp_rpm/plugins/migrations/0031_regenerate_repo_unit_counts.py

```diff
@@ -45,6 +45,6 @@ def migrate(*args, **kwargs):
     """
     db = connection.get_database()
     repos_collection = db['repos']
-    for repo in repos_collection.find():
-        repo_id = repo['repo_id']
+    repo_ids = [x['repo_id'] for x in repos_collection.find()]
```

Comment:
I think you want to limit the fields that you are retrieving here.

http://docs.mongoengine.org/guide/querying.html#query-efficiency-and-performance